action.yml

```diff
@@ -1,13 +1,15 @@
-name: 'Custom Action Template'
+name: 'Is Actor Authorized'
+
+description: 'This action will determine if the actor who initiated the workflow is authorized to do so.'
 
-description: 'Add description here'
 inputs:
-  input-1:
-    description: 'Add a description here'
+  actor: 
+    description: 'The actor who initiated the workflow'
+    required: true
```

Comment:
Not that passing in the actor isn't super simple, but I'm curious if there was a specific reason not to use `@actions/github`'s actor context property. I'm pretty sure that's possible, though I haven't memorized that context.

The only real benefit would be cutting down on an invocation property, making usage slightly easier. Not really a huge deal, like I said, because it's so simple to pass in.

Reply:
I originally had it that way in my actions-testing repo.  I decided to pass it in as an arg so it was super clear about where it was getting that value.  

Like you, I also thought about someone's future edge case where the might want to use it for something other than the person who kicked off the workflow so an arg seemed better for that one use case that might not ever happen.